molo/profiles/templates/profiles/register.html

```diff
@@ -2,50 +2,60 @@
 {% load i18n static core_tags wagtailsettings_tags %}
 {% get_settings %}
 
-
 {% block content %}
-<div class="title profile">
-    <h3>{% trans "Join" %}</h3>
-</div>
-<div class="block">
-  <div class="signin-form">
-    <p>{% trans "Not a member yet? Join us by signing up below." %}</p>
+<h1 class="heading heading--profiles">{% trans "Register" %}</h1>
+<h4 class="heading heading--subtitle">
+    {% trans "Not a member yet?" %}<br/>
+    {% trans "Join us by signing up below." %}
+</h4>
+<div class="profiles">
+  <form method="post" action="{% url "molo.profiles:user_register" %}" class="profiles__form">
+  {% csrf_token %}
+    <fieldset>
+      <label for="id_username">{% trans "Choose a username" %}</label>
+      {{ form.username.errors}}
+      {{ form.username }}
+    </fieldset>
 
-    <form method="post" action="{% url 'molo.profiles:user_register' %}">{% csrf_token %}
-        <p><strong>{% trans "Choose a username" %}</strong></p>
-        {{ form.username.errors}}
-        {{ form.username }}
+    <fieldset>
+      <label for="id_password">{% trans "Choose a 4-digit pin" %}</label>
+      {{ form.password.errors}}
+      {{ form.password }}
+      <p class="sigin-form__input-hint">({% trans "Eg.: 2086" %})</p>
```

Comment:
Please place this in the label, make it `<span class="label__hint">{% trans "Eg: 2086" %}</span>`